DP03_05_Near-Earth_Objects.ipynb

```diff
@@ -0,0 +1,1395 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "80a0baf5-51ad-40ec-8991-060a7b27c289",
+   "metadata": {},
+   "source": [
+    "**Skills:** _Use TAP queries to retrieve Solar System objects. Plot properties and orbits of near-Earth objects._"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "393da88f-7978-4920-aa4a-a9830df6eed9",
+   "metadata": {},
+   "source": [
+    "**LSST Data Products:** _TAP DP0.3 MPCORB (10-year) and SSObject tables._"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5c67fab9-136a-4adc-bb42-142b91ab69dd",
+   "metadata": {},
+   "source": [
+    "**Packages:** _lsst.rsp.get_tap_service_"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8f72b27f",
+   "metadata": {},
+   "source": [
+    "**Credit:**\n",
+    "_Originally developed by Sarah Greenstreet and the Rubin Community Science Team in the context of the Rubin DP0._"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "28e91cbf-ab7f-4e26-9276-b00299d6065e",
+   "metadata": {},
+   "source": [
+    "**Get Support:**\n",
+    "Find DP0-related documentation and resources at <a href=\"https://dp0.lsst.io\">dp0.lsst.io</a>. Questions are welcome as new topics in the <a href=\"https://community.lsst.org/c/support/dp0\">Support - Data Preview 0 Category</a> of the Rubin Community Forum. Rubin staff will respond to all questions posted there."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cfc73be0",
+   "metadata": {},
+   "source": [
+    "## 1. Introduction\n",
+    "\n",
+    "_This tutorial demonstrates how to use TAP to query the MPCORB and SSObject tables. The tables are queried simultaneously and joined on their SSObjectId. The properties of near-Earth objects (NEOs; perihelia $q$<1.3 au) from the DP0.3 catalogs are examined. The NEOs in this sample are restricted to those with semimajor axes $a$<4.0 au, and eccentricities $e$<1.0. The orbital properties of NEOs are examined, including plotting the planet-crossing regions and boundaries that define the four NEO sub-populations: Amors, Apollos, Atens, and Atiras._"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4695fb41-9d89-4d0b-af9c-b57222077199",
+   "metadata": {},
+   "source": [
+    "### 1.1 NEO Dynamical Classes\n",
+    "\n",
+    "Near-Earth objects (NEOs) are traditionally divided into four dynamical sub-populations based on their orbital parameters compared to the orbit of the Earth. The Earth's orbit is nearly circular with a semimajor axis $a$ = 1.0 au and eccentricity $e$ = 0.017. This gives the Earth a perihelion distance $q$ = 0.983 au and an aphelion distance of $Q$ = 1.017 au. The four NEO sub-populations (Amors, Apollos, Atens, and Atiras) are divided using these orbital parameters of the Earth as follows:\n",
+    "\n",
+    "**Amors** have orbits exterior to Earth's orbit with perihelia larger than Earth's aphelion and less than 1.3 au (1.017 au < $q$ < 1.3 au; i.e., Amors are always farther from the Sun than Earth).\n",
```

Comment:
consistent () around i.e. statement?  Or is it connected with the (1.017 au...)

Reply:
() moved for easier reading.